aspnet/migration/rc1-to-rc2.rst

```diff
@@ -76,15 +121,28 @@ IHostingEnvironment changes
 
 All environment variables are now prefixed with the ``ASPNETCORE_`` prefix.
 
-======================  ===========================
-Old prefix              New prefix
-======================  ===========================
-ASPNET\_WEBROOT         ASPNETCORE\_WEBROOT
-ASPNET\_SERVER          ASPNETCORE\_SERVER
-ASPNET\_APP             ASPNETCORE\_APPLICATIONNAME
-ASPNET\_ENVIRONMENT     ASPNETCORE\_ENVIRONMENT
-ASPNET\_DETAILEDERRORS  ASPNETCORE\_DETAILEDERRORS
-======================  ===========================
+===================================================  ===================================================
+RC1                                                  RC2
+===================================================  ===================================================
+ASPNET_APP, ASPNET_APPLICATIONNAME, Hosting:App      ASPNETCORE_APPLICATIONNAME
+ASPNET_STARTUPASSEMBLY                               ASPNETCORE_STARTUPASSEMBLY
+ASPNET_DETAILEDERRORS, Hosting:DetailedErrors        ASPNETCORE_DETAILEDERRORS
+ASPNET_ENVIRONMENT, ASPNET_ENV, Hosting:Environment  ASPNETCORE_ENVIRONMENT
+ASPNET_SERVER, Hosting:Server                        ASPNETCORE_SERVER
+ASPNET_WEBROOT, webroot                              ASPNETCORE_WEBROOT
+ASPNET_CAPTURESTARTUPERRORS                          ASPNETCORE_CAPTURESTARTUPERRORS
+ASPNET_SERVER.URLS                                   ASPNETCORE_SERVER.URLS
+ASPNET_CONTENTROOT, ASPNET_APPLICATIONBASE           ASPNETCORE_CONTENTROOT
+===================================================  ===================================================
+
+
+In RC2, you can use whatever prefix you want. You should add it explicitly by calling:
+
+.. code-block:: c#
+
+  new ConfigurationBuilder.AddEnvironmentVariables(“ANY_PREFIX_YOU_WANT_”).Build(); 
+  
+However, there is an exception. You must set the environment key using ``ASPNETCORE_ENVIRONMENT``. This is picked up by default by the ``WebHostBuilder``, unlike the other variables. 
 
 ASP.NET 5 was renamed to ASP.NET Core 1.0. Also, MVC and Identity are now part of ASP.NET Core. ASP.NET MVC 6 is now ASP.NET Core MVC. ASP.NET Identity 3 is now ASP.NET Core Identity.
```

Comment:
This should be the first sentence in the "Namespace and package ID changes" section. It's too hidden here.